setup/requirements.md

```diff
@@ -43,6 +43,20 @@ currently require the following versions _or newer_:
 If you're using [Remote IDEs](../environments/editors.md), allow pop-ups; Coder
 launches the Remote IDE in a pop-up window.
 
+## Network Policies
+
+Coder uses
+[Container Network Interface (CNI)](https://github.com/containernetworking/cni#what-is-cni)
+plugins to implement network segmentation and tenant isolation in the Kubernetes
+cluster. This enforces network boundaries between pods, preventing users from
+accessing other workspaces.
+
+If there is no CNI plugin configured to enforce such policies, workspaces will
+be able to communicate with each other without restriction.
+
+See more:
```

Comment:
**suggestion**: consider collating these links at the bottom of the page in a "See also" section, or turn this into a "Learn more about Network Policies" link?

Reply:
Good with me.

Reply:
> We may want to specify a recommendation based on what we test (or state that we test with a specific CNI provider, whatever that is), for the people that are unfamiliar with Kubernetes and don't know which CNI provider to choose.

Agreed, I wasn't sure if we wanted to get into recommending CNI providers, but since Calico it is already built into GCP & Azure, I'd say it's accredited enough to recommend here.

Reply:
Yeah, I'm not sure about the idea of recommending things either - the only reason I mentioned it was for the use case of "I want to try Coder but I don't know much about Kubernetes"

We could say "If you're not sure, we suggest choosing Calico" or something to that effect? Any CNI provider should work and it's not a recommendation per se (because we're not saying to choose Calico over Cilium or something else, if your organization standardizes on those).

My intent would just be to nudge users toward whatever we test with, because it's more likely to work

Reply:
Agree - let's go with the "if you're not sure" verbiage, as it isn't a blatant recommendation.